src/navigation/wallet/screens/CreateMultisig.tsx

```diff
@@ -527,36 +527,12 @@ const CreateMultisig = () => {
                         setTestnetEnabled(!testnetEnabled);
                         setOptions({
                           ...options,
-                          networkName: testnetEnabled ? 'testnet' : 'livenet',
+                          networkName: !testnetEnabled ? 'testnet' : 'livenet',
```

Comment:
nit: since we are doing this at more than one place can create a method and call it `onPress={toggleTestnet}`

```
  const toggleTestnet = () => {
    const _testnetEnabled = !testnetEnabled;
    setTestnetEnabled(_testnetEnabled);
    setOptions({
      ...options,
      networkName: _testnetEnabled ? 'testnet' : 'livenet',
    });
  }
```